### DISCUSSION TITLE
Seo URLS and Google

### DISCUSSION THREAD

**Discussion by lumibliss** [Chevereto/Chevereto-Free#151] [Help needed]
Hello,

Iam encountering a strange problem.

I have installed the free version of Chevereto on my domain in 2 different directories.
Directory A and Directory B.

In both sites, the seo for images and album are enabled, and the urls really do get the correct filename.
However, with directory A, google index the site urls correctly, (example: image/cat-food-very-tasty) with the full image name as title.
But with the other directory, the posts are getting indexed with their short url title, like, image/KgLp. even though the post link shows the full picture name)
How come its indexing it wrong? what can I do to change tis behavior? it doesn't make any sense for it to be indexed differently than than directory A.

**Comment by rodber** [comment]
Hi @lumibliss

🤷🏽‍♂️ As the implementation is exactly the same, I think that is safe to believe that the issue is on how Google is crawling your website.
